projects/story-decorator/src/StoryDecorator.ts

```diff
@@ -46,8 +46,8 @@ const urlParams = new URLSearchParams(queryString);
 
 export let dir: 'ltr' | 'rtl' =
     (urlParams.get('sp_dir') as 'ltr' | 'rtl') || 'ltr';
-export let theme: ThemeVariant =
-    (urlParams.get('sp_theme') as ThemeVariant) || 'spectrum';
+export let system: SystemVariant =
+    (urlParams.get('sp_theme') as SystemVariant) || 'spectrum'; // need to dupe?
```

Comment:
Sorry, meant to improve that comment. I couldn't really understand yet what this `urlParam` was for and if I should introduce an `sp_system` version of it. I had a hard time understanding where to draw the line from deprecation to replacement—but for storybook I thought it would be more okay to replace?